src/warnet/cli/graph.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+from typing import List
+
+import click
+from rich import print
+
+from warnet.cli.rpc import rpc_call
+from warnet.utils import DEFAULT_TAG
+
+
+@click.group(name="graph")
+def graph():
+    """Graph commands"""
+
+
+@graph.command()
+@click.argument("params", nargs=-1, type=str)
```

Comment:
lets just require `--n=<integer>` right? there's no other params

Reply:
Yep agree.

Although... if we ever extend it to other graph types which _do_ have more params, I think this is useful to leave in.

Personally, I feel like it's not much code, which does no harm currently, and I'd prefer to leave it as it's written up now for the future?

Easily persuaded that's not the best course of action here though.

Reply:
it's just weird to me to suddenly have an arg without the `--`. And also, I think filename should either be required, or there should be a flag to pipe the output right back in to `network start`. Would anyone need a 100 node graph file printed to the console?